srcts/src/components/sidebar.ts

```diff
@@ -242,6 +266,307 @@ class Sidebar {
     containers.forEach((container) => new Sidebar(container as HTMLElement));
   }
 
+  /**
+   * Initialize sidebar resize functionality.
+   * @private
+   */
+  private _initResizeHandle(): void {
+    if (!this.layout.resizeHandle) {
+      this._createResizeHandle();
+    }
+    this._updateResizeAvailability();
+  }
+
+  /**
+   * Create the resize handle element.
+   * @private
+   */
+  private _createResizeHandle(): void {
+    const handle = document.createElement("div");
+    handle.className = Sidebar.classes.RESIZE_HANDLE;
+    handle.setAttribute("role", "separator");
+    handle.setAttribute("aria-orientation", "vertical");
+    handle.setAttribute("aria-label", "Resize sidebar");
+    handle.setAttribute("tabindex", "0");
+    handle.setAttribute("aria-keyshortcuts", "ArrowLeft ArrowRight Home End");
+    handle.title = "Drag to resize sidebar";
+
+    const indicator = document.createElement("div");
+    indicator.className = "resize-indicator";
+    handle.appendChild(indicator);
+
+    const instructions = document.createElement("div");
+    instructions.className = "visually-hidden";
+    instructions.textContent =
+      "Use arrow keys to resize the sidebar, Shift for larger steps, Home/End for min/max width.";
+    handle.appendChild(instructions);
+
+    // Insert handle into the layout container
+    this.layout.container.appendChild(handle);
+    this.layout.resizeHandle = handle;
+
+    this._attachResizeEventListeners();
+  }
+
+  /**
+   * Attach event listeners for resize functionality.
+   * @private
+   */
+  private _attachResizeEventListeners(): void {
+    if (!this.layout.resizeHandle) return;
+
+    const handle = this.layout.resizeHandle;
+
+    // Mouse events
+    handle.addEventListener("mousedown", this._onResizeStart.bind(this));
+    document.addEventListener("mousemove", this._onResizeMove.bind(this));
+    document.addEventListener("mouseup", this._onResizeEnd.bind(this));
+
+    // Touch events for mobile devices
```

Comment:
Just curious, have you tested on mobile?

Reply:
Yup! I just tested again now on my iPad and tweaked the touch target area to be a little larger.